config.toml

```diff
@@ -0,0 +1,61 @@
+[taker_config]
+# relatively low value for now so that its easier to test without having to wait too much
+# right now only the very brave will try coinswap out on mainnet with non-trivial amounts
+
```

Comment:
We need to separate the config files as they will be read by separate software eventually. 

As we are handling parsing ourselves, it could as well be a custom format `taker.conf` and `maker.conf`. Similar to `bitcoin.conf`.